vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -431,6 +430,7 @@ void BatPublishers::OnRestorePublishersInternal(bool success) {
 
 void BatPublishers::setPublisherMinVisitTime(const uint64_t& duration) { // In seconds
   state_->min_publisher_duration_ = duration;
+  calcScoreConsts(duration);
```

Comment:
we need to update variables on the fly and not only when we restart the browser

Reply:
we should really move these things to an observer for PublisherMinVisitTime change

Reply:
so ledger would save data and call rewards that save was successful and we would trigger observer that would then call back ledger?

Reply:
no, an observer internal to ledger